engine/kv_engine.cpp

```diff
@@ -380,9 +385,22 @@ Status KVEngine::RestoreSortedRecord(DLDataEntry *pmem_data_entry,
         return Status::MemoryOverflow;
       }
       new_hash_offset = (uint64_t)dram_node;
+// RESTORE_SKIPLIST_STRIDE: Select a data entry every 10000 into restored
+// skiplist map for multi-thread restoring large skiplist.
+#define RESTORE_SKIPLIST_STRIDE 10000
```

Comment:
Don't define here, add a "constexpr uint64_t kRestoreSkiplistStride = 10000;" at begin of the source file.